garage/logger/tensorboard_output.py

```diff
@@ -0,0 +1,272 @@
+from os.path import abspath, dirname
+import shutil
+
+import google.protobuf.json_format as json_format
+from jsonmerge import merge
+import numpy as np
+from tensorboard import summary as summary_lib
+from tensorboard.backend.event_processing import (plugin_event_multiplexer as
+                                                  event_multiplexer)
+from tensorboard.plugins.custom_scalar import layout_pb2, metadata
+import tensorflow as tf
+
+from garage import config
+from garage.logger.outputs import LogOutput
+from garage.logger.tabular_input import TabularInput
+from garage.misc.console import mkdir_p
+
+
+class TensorBoardOutput(LogOutput):
```

Comment:
This might be easily replaced by a wrapper around tensorboardX

https://github.com/lanpa/tensorboardX

Reply:
(this can be in a future PR)